### ISSUE TITLE
How to set the time zone of an event?

### ISSUE THREAD

**Issue by raxod502** [nwcell/ics.js#54]
When I put my events into Apple Calendar, they show up as "Time zone: Floating". This makes sense since I didn't provide any time zone to `ics.js` when I created the events. That leaves the question: how _do_ I provide the time zone?

**Comment by Forrest-T** [comment]
This can be done with Date objects (and Date-friendly formatted strings) as described [here](https://www.w3schools.com/js/js_date_formats.asp).
For example, I am writing this at `2018-09-24T22:33:13-07:00`
- 2018-09-24 is the date
- 22:33:13 is the time
- -07:00 is the offset of my time zone relative to UTC time

**Comment by raxod502** [comment]
Maybe I'm missing something, but that doesn't seem to fix the problem. All adding a time zone does is shift the time. It doesn't actually add a time zone, so I would still expect them to show up as "Time zone: Floating".

**Comment by Forrest-T** [comment]
No, I think I might have misunderstood your question. The above can be used to specify a local time in some time zone (i.e. with some offset), which can then be interpreted as "absolute time" when you import the events. I believe Google Calendar, for example, translates events to whatever local time zone you import from (or has an option to do so). On closer inspection, you are probably looking for specifying the time zone for the calendar app using [VTIMEZONE](https://www.kanzaki.com/docs/ical/vtimezone.html), which unfortunately seems to be [unimplemented](https://github.com/nwcell/ics.js/blob/981e460c1d70cd9267f498d08608bc85e90fffb6/ics.js#L109) right now :-(
